src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -327,6 +348,15 @@ TrackPointer BansheePlaylistModel::getTrack(const QModelIndex& index) const {
     return pTrack;
 }
 
+TrackId BansheePlaylistModel::getTrackId(const QModelIndex& index) const {
```

Comment:
This is identical with the default implementation in BaseExternalPlaylistModel. Obsolete?

Reply:
BansheePlaylistModel does not inherit BaseExternalPlaylistModel.
We fetch the Banshee database directly.

Reply:
:see_no_evil: